pull_request_template.md

```diff
@@ -3,9 +3,13 @@ Write a detailed description of your PR here.
 Please follow the guidelines for PRs contained [here](docs/source/contributing.md). Checklist:
 
 - [ ] Ensure that your PR is small, and implements one change.
+- [ ] Give your PR title one of the prefixes FIX, ENH, STYLE, DOC, DEL to indicate what type of change that is.
+- [ ] Link the correct GitHub issue for tracking.
 - [ ] Add unit tests for all functions that you introduced or modified.
-- [ ] Run PyCharm's code cleanup tools on your Python files.
+- [ ] Run automatic code formatting / linting on all files.
 - [ ] Ensure that documentation renders correctly in Sphinx (run Sphinx via `make html` in the `docs folder)
-- [ ] Link the correct GitHub issue for tracking.
-- [ ] When merging your PR, **replace the default merge message** with a brief description of your PR,
-and if needed a motivation why that change was required.
\ No newline at end of file
+
+## Change the default merge message
+
+When completing your PR, you will be asked for a title and an optional extended description. By default, the extended description will be a concatenation of the individual
```

Comment:
Should the checklist of the description be also removed? Is it a checklist for the developer or for the maintainers?

Reply:
Checklist is more for the developer. I don't care if it stays there as the first comment of the PR

Reply:
@javier-alvarez seems to favour removing it after submitting the PR: https://github.com/microsoft/InnerEye-DeepLearning/pull/678#pullrequestreview-897919818